pydot.py

```diff
@@ -777,13 +965,16 @@ def parse_node_ref(self, node_str):
 
         node_port_idx = node_str.rfind(':')
 
-        if (node_port_idx > 0 and node_str[0] == '"' and
-                node_str[node_port_idx-1] == '"'):
+        if (
+            node_port_idx > 0
+            and node_str[0] == '"'
+            and node_str[node_port_idx - 1] == '"'
+        ):
             return node_str
 
         if node_port_idx > 0:
             a = node_str[:node_port_idx]
-            b = node_str[node_port_idx+1:]
+            b = node_str[node_port_idx + 1 :]
```

Comment:
Causes `E203 whitespace before ':'` from `flake8`/`pycodestyle`, but problem seems to be with them rather than with `black`. See:

- https://github.com/psf/black/blob/21.5b1/docs/the_black_code_style/current_style.md#slices
- https://github.com/PyCQA/flake8/issues/257
- https://github.com/PyCQA/pycodestyle/issues/373
- https://github.com/PyCQA/pycodestyle/pull/914